SEED-platform/seed
# Mapping on import takes into account selected units

#### Any background context you want to provide?
During import, mapping with non-default units was ignored.

#### What's this PR do?
Fixes that problem by creating "raw" Column objects that have a `units_pint` value (previously, it was always `None`) during mapping save. It turns out that this is enough of a fix since, during `perform_mapping`, "cleaners" are built using those Column objects, and subsequently, they use the `units_pint` values for conversions.

Prior, with `units_pint` being None, incoming values defaulted to the base units of the respective fields.

#### How should this be manually tested?
Import records; map to unit-aware columns; choose non-default units; finally, see that the values have been converted according to your selection on the mapping confirmation page.

#### What are the relevant tickets?
#2344 

#### Screenshots (if appropriate)
